src/Fhir.Anonymizer.Shared.Core.UnitTests/Visitors/AnonymizationVisitorTests.cs

```diff
@@ -476,5 +555,17 @@ private Person CreateTestPerson()
 
             return person;
         }
+
+        private Observation CreateTestObservation()
+        {
+            Observation observation = new Observation();
+            observation.ReferenceRange.Add(
+                new Observation.ReferenceRangeComponent
+                {
+                    Low = new SimpleQuantity { Value = 10 },
+                    High = new SimpleQuantity { Value = 100},
+                });
+            return observation;
```

Comment:
should we add other elements in observation.referenceRange.low？
To test:  If we perturb "observation.referenceRange.low" and then redact it, other children of "low" are still there even if only value will be changed.

Reply:
Thanks. Added